packages/cashier/src/Components/Receipt/payment-agent-receipt.jsx

```diff
@@ -37,26 +36,33 @@ class PaymentAgentReceipt extends React.Component {
                         </span>
                     </div>
                     <div className='cashier__transferred-details-wrapper'>
-                        <Icon icon={`IcCurrency-${currency_lowercase}`} />
                         <span className='cashier__transferred-details'>
-                            <span className='cashier__text--bold'>{getCurrencyDisplayCode(currency)}</span>
-                            &nbsp;(
-                            {loginid})
+                            <span className='cashier__text--bold cashier__text--end'>
+                                {getCurrencyDisplayCode(currency)}
+                            </span>
```

Comment:
```suggestion
                            <Text weight="bold" className='cashier__text--end'>
                                {getCurrencyDisplayCode(currency)}
                            </Text>
```

Reply:
I have noticed these inconsistencies in the code which has been ignored in their respective card/issue. I was hoping we could fix them in a separate/more related PR, but since you've mentioned it here then i will proceed to update.